test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
I think you could programmatically mock _all_ of the `fs` methods in a loop. Just check if the first argument passed to the method is a string, and if so replace it with the temp filename, then call original `fs` method. It appears most of the `fs` methods take a `path` as their first argument.